src/systemd/sd-dhcp-server.h

```diff
@@ -85,6 +85,9 @@ int sd_dhcp_server_forcerenew(sd_dhcp_server *server);
 
 int sd_dhcp_server_is_in_relay_mode(sd_dhcp_server *server);
 int sd_dhcp_server_set_relay_target(sd_dhcp_server *server, const struct in_addr* address);
+int sd_dhcp_server_set_agent_circuit_id(sd_dhcp_server *server, const char* agent_circuit_id);
+int sd_dhcp_server_set_agent_remote_id(sd_dhcp_server *server, const char* agent_remote_id);
```

Comment:
Maybe, `sd_dhcp_server_set_relay_agent_circuit/remote_id()` is better for consistency with `RelayAgentCircuit/RemoteId=`.